test/squish/suite_nuke_submitter/run_squish_test.py

```
@@ -18,11 +19,8 @@ def run_squish_command(testcase_name, local=True):
             - success (bool): True if command executed successfully (return code 0)
             - job_id (str or None): Extracted job ID from output if found, None otherwise
     """
-    deadline_nuke_path = os.environ.get("DEADLINE_NUKE_PATH")
-    if not deadline_nuke_path:
-        print("DEADLINE_NUKE_PATH environment variable not set")
-        return False, None
```

If we're removing this, do we have any handling elsewhere where the tests will fail early if DEADLINE_NUKE_PATH is not set?

---

Yes, `test_valid_environment_variables` tests to make sure that all the environment variables needed are set.